src/azul/openapi/spec.py

```diff
@@ -0,0 +1,221 @@
+                    reachable via HTTP(S) but nothing more.
+                '''),
+                **self._health_spec(self._up_key)
+            }
+        }
+
+    def cached_health(self) -> JSON:
+        return {
+            'method_spec': {
+                'summary': 'Cached health check for continuous monitoring',
+                'description': format_description('''
+                    Return a cached copy of the
+                    [`/health/fast`](#operations-Auxiliary-get_health_fast) response.
+                    This endpoint is optimized for continuously running, distributed health
+                    monitors such as Route 53 health checks. The cache ensures that the
+                    service is not overloaded by these types of health monitors. The cache
+                    is updated every minute.
+                '''),
+                **self._health_spec(self._fast_keys)
+            }
+        }
+
+    def fast_health(self) -> JSON:
+        return {
+            'method_spec': {
+                'summary': 'Fast health check',
+                'description': format_description('''
+                    Performance-optimized health check of the REST API and other critical
+                    resources the service depends on. This endpoint can be requested more
+                    frequently than [`/health`](#operations-Auxiliary-get_health) but
+                    periodically scheduled, automated requests should be made to
+                    [`/health/cached`](#operations-Auxiliary-get_health_cached).
+                '''),
+                **self._health_spec(self._fast_keys)
+            }
+        }
+
+    def custom_health(self) -> JSON:
+        return {
+            'method_spec': {
+                'summary': 'Selective health check',
+                'description': format_description('''
+                    This endpoint allows clients to request a health check on a specific set
+                    of resources. Each resource is identified by a *key*, the same key
+                    under which the resource appears in a
+                    [`/health`](#operations-Auxiliary-get_health) response.
+                '''),
+                **self._health_spec(self._all_keys)
+            },
+            'path_spec': {
+                'parameters': [
+                    params.path(
+                        'keys',
+                        type_=schema.array(schema.enum(*sorted(Health.all_keys))),
+                        description='''
+                            A comma-separated list of keys selecting the health checks to be
+                            performed. Each key corresponds to an entry in the response.
+                        ''')
+                ]
+            }
+        }
+
+    def openapi(self) -> JSON:
+        return {
+            'method_spec': {
+
+                'summary': 'Return OpenAPI specifications for this service',
```

Comment:
```suggestion
                'summary': 'Return OpenAPI specifications for this REST API',
```

Please check if there are more of these cases, where documentation isn't general enough or needs to be parameterized.